cypress/e2e/auth.cy.js

```diff
@@ -0,0 +1,104 @@
+const USER_NAME = "CypressJester";
+const USER_EMAIL = "crypressJest@noroff.no";
+const USER_PASSWORD = "12345678";
+const USER_AVATAR =
+  "https://images.pexels.com/photos/1769271/pexels-photo-1769271.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2";
+
+describe("Authorization", () => {
+  beforeEach(() => {
+    cy.wait(500);
+    cy.visit("/");
+  });
+
+  it("Cannot view profile page", () => {
+    cy.visit("/?view=profile");
+  });
+
+  it("Cannot view post page", () => {
+    cy.visit("/?view=posts");
+  });
+
+  it("Can register with valid credentials", () => {
+    cy.get("form#registerForm").wait(500);
+    cy.get(`input[name="name"]`).type(USER_NAME);
+    cy.get(`input#registerEmail[name="email"]`).type(USER_EMAIL);
+    cy.get(`input#registerPassword[type="password"]`).type(USER_PASSWORD);
+    cy.get(`input#registerAvatar[name="avatar"]`).type(USER_AVATAR);
+    cy.get(`button[type="submit"]`).contains("Create Profile").click();
```

Comment:
Something should happen next here too, like checking the URL is correct for a single post